elastalert/alerts.py

```diff
@@ -320,12 +322,22 @@ def __init__(self, rule):
         self.jira_args = {'project': {'key': self.project},
                           'issuetype': {'name': self.issue_type}}
 
+        if self.security_level:
+            self.jira_args['security'] = {'name': self.security_level}
+
         if self.component:
             self.jira_args['components'] = [{'name': self.component}]
         if self.label:
             self.jira_args['labels'] = [self.label]
         if self.assignee:
             self.jira_args['assignee'] = {'name': self.assignee}
+        # Note: because of the sometimes strange behavior of JIRA with custom field types, we have
```

Comment:
It shouldn't be a whole lot of work to hide the customfield_1234 exposure to the end-users.  There is a rest endpoint that JIRA exposes which gives all the custom field mappings:

`
>>> import requests
>>> r = requests.get('https://jira/rest/api/2/field')
>>> r.text
u'[{"id":"customfield_12013","name":"Report Type","custom":true,"orderable":true,"navigable":true,"searchable":true,"clauseNames":["cf[12013]","Report Type"],"schema":{"type":"string","custom":"com.atlassian.jira.plugin.system.customfieldtypes:select","customId":12013}}
,{"id":"fixVersions","name":"Fix Version/s","custom":false,"orderable":true,"navigable":true,"searchable":true,"clauseNames":["fixVersion"],"schema":{"type":"array","items":"version","system":"fixVersions"}},{"id":"customfield_11200","name":"Rank","custom":true,"orderab
le":true,"navigable":true,"searchable":true,"clauseNames":["cf[11200]","Rank"],"schema":{"type":"array","items":"string","custom":"com.pyxis.greenhopper.jira:gh-lexo-rank","customId":11200}},{"id":"customfield_12015","name":"SR/MTG #","custom":true,"orderable":true,"nav
igable":true,"searchable":true,"clauseNames":["cf[12015]","SR/MTG #"],"schema":{"type":"string","custom":"com.atlassian.jira.plugin.system.customfieldtypes:textfield","customId":12015}},{"id":"resolution","name":"Resolution","custom":false,"orderable":true,"navigable":t
rue,"searchable":true,"clauseNames":["resolution"],"schema":{"type":"resolution","system":"resolution"}},{"id":"customfield_11201","name":"Team Members","custom":true,"orderable":true,"navigable":true,"searchable":true,"clauseNames":["cf[11201]","Team Members"],"schema"
:{"type":"array","items":"user","custom":"com.atlassian.jira.plugin.system.customfieldtypes:multiuserpicker","customId":11201}},{"id":"customfield_12017","name":"Google Doc[...]`

Havingsaidthat, there's probably some edge cases and other things that you'd have to deal with to really pull it off correctly.  

This pull request has been outstanding for quite some time.  Have you been using this and has it been working for you, @banjoey ?